src/components/_classes/component/Component.unit.js

```diff
@@ -47,7 +47,7 @@ describe('Component', () => {
       bad: {
         value: 't',
         field: 'firstName',
-        error: 'First Name must be longer than 1 characters.'
+        error: 'First Name must be at least 2 characters.'
```

Comment:
Also change this to "must have" so it is consistent with the minimum words messages.